circle.yml

```diff
@@ -89,35 +89,19 @@ jobs:
           name: Install build dependencies
           command: |
             apt-get -qq update
-            apt-get -qy install ccache cmake libboost-all-dev libz3-dev
+            apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libz3-dev
```

Comment:
Is it faster if we just install the necessary ones compared to `boost-all`? I'm fine if that is the motivation.

Reply:
Yes, much faster.